apps/vaporgui/PButton.cpp

```diff
@@ -0,0 +1,13 @@
+#include "PButton.h"
+#include "VPushButton.h"
+
+PButton::PButton(std::string label, Callback cb)
+: PWidget("", _button = new VPushButton(label)), _cb(cb)
```

Comment:
This seems to be a new form of PWidget that uses a callback, not the HLI, or (necessarily) the LLI.  Can you talk about what was insufficient about the LLI/HLI method we use everywhere else?

Reply:
The LLI and HLI are used to sync a widget with the params database. Given that this is a button, it is not synced with the params database so it does not make sense to use those functions.